lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -49,7 +49,8 @@ type ICache struct {
 }
 
 const (
-	imageCacheKey = "images"
+	imageCacheKey     = "images"
+	defaultHashPrefix = "sha256:"
```

Comment:
Minor: you could use docker's [`digest.Canonical`](https://github.com/vmware/vic/blob/a5383fd8fdbfbe118b780e8e8952622a787ffb32/vendor/github.com/docker/distribution/digest/digester.go#L17) here.

Reply:
Good call, will fix. I'll also update the reference in `backends/image.go`.